schemas/spdx-schema.json

```diff
@@ -343,7 +345,9 @@
             "type" : "boolean"
           },
           "hasFiles" : {
-            "description" : "Indicates that a particular file belongs to a package.",
+            "description" : "DEPRECATED: use relationships instead of this field. Indicates that a particular file belongs to a package.",
+            "deprecated": true,
+            "$comment": "This field has been deprecated as it is a duplicate of using SPDXRef-<package-id> CONTAINS SPDXRef-<file-id> relationships",
```

Comment:
nit: for consistency, maybe just SPDXRef-PACKAGE CONTAINS?

Reply:
If you're referring to the `SPDXRef-DOCUMENT` for the other deprecation text, I don't think there is a consistency problem because that is the actual value, whereas the `SPDXRef-<package-id>` needs to match the containing package, but I'm happy to change this if people feel strongly. I'll see if I can find a better way to identify this.

Reply:
I did update the text here a bit, hopefully it's clear enough.